### DISCUSSION TITLE
期待出现储存策略到储存策略的附件迁移功能

### DISCUSSION THREAD

**Discussion by iisyw** [halo-dev/halo#6068] [Ideas]
### 你当前使用的版本

2.16.1

### 描述一下此特性

之前我迁移博客并没有设置储存策略，导致附件全在upload文件夹内

现在只有附件组到附件组的附件迁移，期待出一个可以批量迁移附件到另外的储存策略的功能，迁移到外部储存策略可能不太好弄，本地对本地的储存策略附件迁移不知道会不会好弄一些

期待功能：本地对本地储存策略中的附件迁移，并且迁移后，自动更新文章内引入附件的路径

**Comment by ruibaby** [comment]
期望能够通过插件实现。

/area plugin
